specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/CommonDefinitions.json

```diff
@@ -2389,12 +2393,23 @@
           "type": "integer"
         },
         "ipSecurityRestrictions": {
-          "description": "IP security restrictions.",
+          "description": "IP security restrictions for main.",
           "type": "array",
           "items": {
             "$ref": "#/definitions/IpSecurityRestriction"
           }
         },
+        "scmIpSecurityRestrictions": {
+          "description": "IP security restrictions for scm.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IpSecurityRestriction"
+          }
+        },
+        "scmIpSecurityRestrictionsUseMain": {
```

Comment:
bools are typically not recommended since they are less descriptive and dont allow for future expansion of values in case its needed.

Reply:
@suwatch - Any response on this comment?